docs/configuration/configuration.md

```diff
@@ -1350,9 +1350,12 @@ See below for the configuration options for Kubernetes discovery:
 # One of endpoints, service, pod, node, or ingress.
 role: <string>
 
+# Optional kube_config will use kube_config file to connect to API server
+# `kube_config` the filepath of kubeconfig, default located at `~/.kube/config`
```

Comment:
There is no real default, because by default we would use the in-cluster configuration.

Reply:
ok, I will fix it